### PR TITLE
EN6-136 cms widgets

### PR DESCRIPTION
**Pull request by ichalagashvili**
Import cms widgets into app-builder

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/widgets/config/WidgetConfigForm.js [hunk 1]
```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { intlShape } from 'react-intl';
 import Form from 'react-jsonschema-form';
+import apps from 'entando-apps';
 
-const WidgetConfigForm = ({ onSubmit, widgetConfig }) =>
-  (widgetConfig && widgetConfig.schema ? (
+const widgetForms = apps.reduce((obj, app) => ({
```
Comment (gidesan) [inline src/ui/widgets/config/WidgetConfigForm.js]: This shouldn't be coupled with CMS. It should be an integration point for every kind of appbuilder app

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by ichalagashvili** [comment]
- With current approach if two different apps have widget config form with same ids, then one will overwrite another. To fix that I think its good if we pass `app.id` too inside `putPageWidget` API call, then during reading the widget config from BE, we will be able to distinguish which config belongs to which app's widget form. @gidesan

**Comment by gidesan** [comment]
> * With current approach if two different apps have widget config form with same ids, then one will overwrite another. To fix that I think its good if we pass `app.id` too inside `putPageWidget` API call, then during reading the widget config from BE, we will be able to distinguish which config belongs to which app's widget form. @gidesan

Makes sense!

**Pull request merged**